GoRogue/MapGeneration/Region.cs

```diff
@@ -9,17 +10,56 @@
 
 namespace GoRogue.MapGeneration
 {
+    /// <summary>
+    /// Event fired when a region's Area is changed.
+    /// </summary>
+    [PublicAPI]
+    public class RegionAreaChanged : EventArgs
```

Comment:
```suggestion
    public class RegionAreaChangedEventArgs : EventArgs
```

By convention from other parts of the library.